vc_zoom/indico_vc_zoom/api/client.py

```diff
@@ -0,0 +1,243 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2020 CERN and ENEA
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from __future__ import absolute_import, unicode_literals
+
+import time
+
+import jwt
+from requests import Session
+from requests.exceptions import HTTPError
+from pytz import utc
+
+
+def format_iso_dt(d):
+    """Convert a datetime objects to a UTC-based string.
+
+    :param d: The :class:`datetime.datetime` to convert to a string
+    :returns: The string representation of the date
+    """
+    return d.astimezone(utc).strftime("%Y-%m-%dT%H:%M:%SZ")
+
+
+def _handle_response(resp, expected_code=200, expects_json=True):
+    resp.raise_for_status()
+    if resp.status_code != expected_code:
+        raise HTTPError("Unexpected status code {}".format(resp.status_code), response=resp)
+    if expects_json:
+        return resp.json()
+    else:
+        return resp
```

Comment:
```suggestion
    return resp.json() if expects_json else resp
```

However, if we never use the response in case of not expecting JSON, maybe returning `None` would be cleaner since that way it's perfectly clear that the response is not used at all?